pkg/version/version.go

```diff
@@ -143,7 +156,7 @@ func (i *Info) IsValidUpgradePath(targetVer string) (ok bool, failureReason stri
 		// version of vertica that came out after the version of this operator.
```

Comment:
Since our release versions are more predictable, we can generalize this logic better. Basically if we are on x.y.0, if y < 4 we can go to x.4.0, if y == 4, we can go to x+1.4.0.

Reply:
Yes, I guess the map really isn't needed anymore for new releases.